src/wallet/wallet.cpp

```diff
@@ -1823,6 +1893,14 @@ bool CWallet::CreateTransaction(const vector<CRecipient>& vecSend,
                     strFailReason = _("Transaction too large");
                     return false;
                 }
+                
+                //remove signature if we used the signing only for the fee calculation
```

Comment:
Why sign at all if it's only for the fee calculation? I ask because we don't want to unnecessarily require the signing keys.